### PR TITLE
Aerospike content update

### PR DESCRIPTION
**Pull request by volmarl**

### CODE REVIEW

**Comment by volmarl** [comment]
@tianon Do the content.md get merge on a schedule?

File: aerospike/content.md [hunk 1]
```diff
@@ -1,82 +1,187 @@
...
 
-# Using this Image
+Anyone can [sign up](https://www.aerospike.com/lp/try-now/) to get an evaluation feature key file for a full-featured, single-node Aerospike Enterprise Edition.
 
-The following will run `asd` with all the exposed ports forwarded to the host machine.
+### Running a node with a feature key file in a mapped directory
 
 ```console
-$ docker run -d --name aerospike -p 3000:3000 -p 3001:3001 -p 3002:3002 -p 3003:3003 %%IMAGE%%
+docker run -tid -v DIR:/opt/aerospike/etc/ -e "FEATURE_KEY_FILE=/opt/aerospike/etc/features.conf" --name aerospike -p 3000:3000 -p 3001:3001 -p 3002:3002 aerospike/aerospike-server-enterprise
```
Comment (yosifkit) [inline aerospike/content.md:35]: This should reference the `ee` image, right?   I don't think `-ti` should be added absolutely unless it is necessary (and I haven't seen any software that requires a tty unless it is interactive like a shell).

(Same changes should be applied in a few other places below.)

```suggestion
docker run -d -v DIR:/opt/aerospike/etc/ -e "FEATURE_KEY_FILE=/opt/aerospike/etc/features.conf" --name aerospike -p 3000:3000 -p 3001:3001 -p 3002:3002 %%IMAGE%%:ee-5.5.0.9
```

Though maybe more generic to not reference a specific version:

```suggestion
docker run -d -v DIR:/opt/aerospike/etc/ -e "FEATURE_KEY_FILE=/opt/aerospike/etc/features.conf" --name aerospike -p 3000:3000 -p 3001:3001 -p 3002:3002 %%IMAGE%%:ee-[version]
```

**Review by yosifkit — COMMENTED** [review]
A suggestion below to adjust the `docker run` examples.

The new doc focuses only on the enterprise edition, but doesn't make clear what options are also available on the community edition (or maybe I missed it). I think it would help users to know which configuration options (like available environment variables) are enterprise only (or that everything but `FEATURE_KEY_FILE` is available).

Maybe also a link to the feature matrix (https://www.aerospike.com/products/product-matrix/) would be useful for users?

File: aerospike/content.md (hunk 1 above)
Comment (volmarl) [reply aerospike/content.md:35]: good points on the identifying which env are EE vs. CE and updating docker run and using ee tag with [version] placeholder.  Circling back internally on this.

**Comment by volmarl** [comment]
@yosifkit implemented your suggestions. thx.

**Comment by yosifkit** [comment]
Should swap `-tid` to `-d` on lines 44, 60, 95, 105, 124, 144, and 152 since a tty is not required.  Should also swap `aerospike/aerospike-server-enterprise` to `%%IMAGE%%:ee-[version]` on the same lines to point users to the new `ee-*` image.

**Comment by volmarl** [comment]
@yosifkit Made the suggested changes. thx

**Pull request merged**